src/extension.ts

```diff
@@ -81,7 +81,8 @@ export function activate(context: vscode.ExtensionContext): void {
             node = <IAzureNode<WebAppTreeItem>>await tree.showNodePicker(WebAppTreeItem.contextValue);
         }
 
-        node.openInPortal();
+        node.treeItem.contextValue === 'deploymentSlot' ? node.openInPortal(`${node.parent.parent.treeItem.id}/slots/${node.treeItem.label}`) : node.openInPortal();
```

Comment:
I believe `node.treeItem.id` would work (which is different than `node.id`)? If that doesn't work - use siteWrapper.id somehow rather than manually constructing the id